assets/script.js

```diff
@@ -111,7 +179,7 @@ $("#searchContent").on("click", function (event) {
 })
 
 //Saves the event to local storage and append event to saved links
-$("#eventData").on("click", ".save-event", function (event) {
+$("#innerData").on("click", ".save-event", function (event) {
```

Comment:
You'll have to switch this back to $("#eventData") so that the event listener works for all save buttons